### DISCUSSION TITLE
MS Paint crashed with ExplorerPatcher

### DISCUSSION THREAD

**Discussion by Martin63500** [valinet/ExplorerPatcher#2981] [General]
Hi,
when i use MSPAINT, it is crashed. I see the following infos in the eventvwr:

Name der fehlerhaften Anwendung: mspaint.exe, Version: 11.2311.30.0, Zeitstempel: 0x65aa4660
Name des fehlerhaften Moduls: ExplorerPatcher.amd64.dll, Version: 22621.2861.62.2, Zeitstempel: 0x65858f42
Ausnahmecode: 0xc0000005
Fehleroffset: 0x000000000000d168
ID des fehlerhaften Prozesses: 0x0xA710
Startzeit der fehlerhaften Anwendung: 0x0x1DA739AF8273882
Pfad der fehlerhaften Anwendung: C:\Program Files\WindowsApps\Microsoft.Paint_11.2311.30.0_x64__8wekyb3d8bbwe\PaintApp\mspaint.exe
Pfad des fehlerhaften Moduls: C:\Program Files\ExplorerPatcher\ExplorerPatcher.amd64.dll
Berichtskennung: c7423f62-825c-4efc-bcd8-3f77ebb268ca
Vollständiger Name des fehlerhaften Pakets: Microsoft.Paint_11.2311.30.0_x64__8wekyb3d8bbwe
Anwendungs-ID, die relativ zum fehlerhaften Paket ist: App

Can you fix it?

Bye
Martin

**Comment by Amrsatrio** [comment]
Please disable "Register as shell extension" in EP Properties > File Explorer.
